scripts/location_utils.py

```diff
@@ -21,16 +24,17 @@ def get_application_form(app_json_blob):
     ]
 
 
-def get_postcode_from_questions(form_questions) -> str:
+def get_postcode_from_questions(form_questions, fundround) -> str:
```

Comment:
This method is now duplicating what you've got in `derive_application_values` - can we just have one implementation?

Reply:
It's not duplicating. In `derive_application_values` it is using entire json and here form_questions is only part of application which is also not a json.  so can't reuse  the code